### PR TITLE
[Delivers #100955526] allow anyone who can see the Proposal to add other observers

### PR DESCRIPTION
**Pull request by afeld**
We might want to tighten this up a bit down the road, but I think it's fine for now.

### CODE REVIEW

File: app/controllers/observations_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class ObservationsController < ApplicationController
   before_action :authenticate_user!
   before_action :find_proposal
-  before_action ->{authorize @proposal, :can_edit!}
+  before_action ->{authorize @proposal, :can_show!}
```
Comment (afeld) [inline app/controllers/observations_controller.rb]: Wondering if it's worthwhile to break out a separate ObservationPolicy...

**Comment by cmc333333** [comment]
Double checking, do https://github.com/18F/C2/blob/master/app/controllers/observations_controller.rb#L4 and https://github.com/18F/C2/blob/master/app/policies/ncr/work_order_policy.rb#L11 make this change not required?

**Comment by afeld** [comment]
Good point, though it sounds like we want to make observers add-able more broadly.

https://www.pivotaltracker.com/story/show/99421886

**Comment by cmc333333** [comment]
Perfect; this'll do for now, then

**Pull request merged**